test.js

```diff
@@ -102,4 +102,23 @@ describe('dat-gateway', function () {
       throw e
     })
   })
+
+  it('should open a websocket to /peers', function () {
+    const req = http.get('http://localhost:5917/bunsen.hashbase.io/')
+    req.on('error', console.log)
```

Comment:
You should get rid of the error handler so it can fail if something unexpected happens.